adagrams/game.py

```diff
@@ -1,11 +1,75 @@
+import random
 def draw_letters():
-    pass
+    '''returns a list of ten random letters as strings from a letter frequency pool'''
+    letters = ['A','A','A','A','A','A','A','A','A','B','B','C','C','D','D','D','D','E','E','E','E','E','E','E','E','E','E','E','E','F','F','G','G','G','H','H','I','I','I','I','I','I','I','I','I','J','K','L','L','L','L','M','M','N','N','N','N','N','N','O','O','O','O','O','O','O','O','P','P','Q','R','R','R','R','R','R','S','S','S','S','T','T','T','T','T','T','U','U','U','U','V','V','W','W','X','Y','Y','Z']
+    return random.sample(letters, 10)
```

Comment:
Smart solution! Because the letter pool never changes, consider making it a constant variable.